tex/generic/pgf/systemlayer/pgfsys-dvips.def

```diff
@@ -138,6 +150,28 @@
 
 \fi
 
+%
+% Transparency groups
+%
+\def\pgfsys@transparencygroupfrombox#1{%
+  \special{ps:
+    mark /NamespacePush pdfmark % limit `{pgfxform}' name visibility
+    mark /_objdef {pgfxform} /BBox [clippath pathbbox newpath] /BP pdfmark
+    save gsave
+    /.setstrokeconstantalpha where
+      {pop 1 .pgfsetfillopacityalpha 1 .pgfsetstrokeopacityalpha} if
+  }%
+  \hskip\pgf@picminx\rlap{\smash{\box#1}}%
```

Comment:
Relying on the definition of `\rlap` and `\smash` is maybe not a good idea.  For example `amsmath` modifies the definition of `\smash`.
```suggestion
  \ht#1=0pt
  \dp#1=0pt
  \hskip\pgf@picminx\hbox to 0pt{\box#1\hss}%
```